packages/model-viewer/src/features/staging.ts

```diff
@@ -63,6 +64,9 @@ export const StagingMixin = <T extends Constructor<ModelViewerElementBase>>(
     @property({type: String, attribute: 'rotation-per-second'})
     rotationPerSecond: string = 'auto';
 
+    @property({type: Number, attribute: 'interpolation-decay-milliseconds'})
+    interpolationDecayMilliseconds: number = DEFAULT_DECAY_MILLISECONDS;
```

Comment:
I don't think this will work, since the attributes from all of the mixins are surfaced on the element and this will create a name collision. Also, I don't see the point of specifying them separately anyway; let's just set both controls and scene from the controls mixin.

Reply:
Done.